src/input.ts

```diff
@@ -246,3 +216,64 @@ export class Mouse {
     await this.click(x, y, { ...options, clickCount: 2 });
   }
 }
+
+const aliases = new Map<string, string[]>([
+  ['ShiftLeft', ['Shift']],
+  ['ControlLeft', ['Control']],
+  ['AltLeft', ['Alt']],
+  ['MetaLeft', ['Meta']],
+  ['Enter', ['\n', '\r']],
```

Comment:
Let's do the same with `description.text` as `description.key`, and remove the `\r` here. Then we are left with just one line adding `\n`.

Reply:
I don't get this one either :/